Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2575,6 +2575,65 @@ def to_context(self):
             return ret_value
 
 
+class IndicatorsTimeline:
+    """
+    IndicatorsTimeline class - use to return Indicator Timeline object to CommandResults
+    :type indicators: ``list`` or ``dict``
+
+    :param indicators: expects a list of indicators or a list of dict of indicators.
+    :type category: ``str``
+
+    :param category: indicator category.
+    :type message: ``str``
+
+    :param message: indicator message.
+    :return: a list of indicators
+
+    :return: None
+    :rtype: ``None``
+    """
+    def __init__(self, indicators=None, category=None, message=None):
```

Comment:
the interface here is not clear. because indicators could contain strings or dicts.
I would suggest CommandResults to have `List[IndicatorsTimeline]`
`indicators` will contain only list of strings. indicators and message should be mandatory params.
If some user want to have message per indicator, he will create list of IndicatorTimeline objects (rare case).